activity.py

```diff
@@ -24,8 +24,8 @@ def __init__(self, handle):
         self.actividad = main.game()
         self.build_toolbar()
         self._pygamecanvas = sugargame.canvas.PygameCanvas(self,
-            main= self.actividad.make,
-            modules= [pygame, pygame.display, pygame.mixer])
+            main= self.actividad.make(),
```

Comment:
The function that's usually passed to `main` isn't called at the time it's passed, could you also fix the spaces after the assignment?

Reply:
> The function that's usually passed to main isn't called at the time it's passed,

I didn't completely understand, could you please give more info on this?

Reply:
You've called `make()` on line 27 while passing it to `main` and usually the functions that get passed to main are referenced and not called when they're being passed.

Reply:
@chimosky, I'm not able to fix this. Could you please help with it?

Reply:
@quozl can you please take a look at this? I haven't figured out why the game canvas doesn't render like it should.

Reply:
Okay, looking into it.